cli/search/search.go

```diff
@@ -0,0 +1,62 @@
+package search
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/apex/log"
+)
+
+// isDeprecated returns if version is greater than 1.9.0.
+func isDeprecated(version string) bool {
+	splitedVersion := strings.Split(version, ".")
+	if len(splitedVersion) < 2 {
+		return false
+	}
+	if splitedVersion[0] == "1" && len(splitedVersion[1]) < 2 {
+		return true
+	}
+	return false
+}
+
+// SearchVersions outputs available versions of program.
+func SearchVersions(programm string) error {
+	cmd := exec.Command("foobar")
+	if programm == "tarantool" {
+		cmd = exec.Command("bash", "-c", "git ls-remote --tags --sort="+
+			"v:refname https://github.com/tarantool/tarantool.git | cut --delimiter='/' --fields=3")
+	} else if programm == "tt" {
+		cmd = exec.Command("bash", "-c", "git ls-remote --tags --sort="+
+			"v:refname https://github.com/tarantool/tt.git | cut --delimiter='/' --fields=3")
+	} else {
+		return fmt.Errorf("Search supports only tarantool/tt")
+	}
+	log.Warn("Available versions of " + programm + ":")
```

Comment:
There are no line breaks between logically different actions, at least separate the data collection and their processing with a new line.